cupy/core/core.pyx

```diff
@@ -1726,10 +1726,20 @@ if (in1 == 0) {
         inv_x = y;
     }
 }
-out0 = in0_type(round(in0.real() * x) * inv_x,
-                round(in0.imag() * x) * inv_x);'''
-
-
+out0 = in0_type(rint(in0.real() * x) * inv_x,
+                rint(in0.imag() * x) * inv_x);'''
```

Comment:
Analagous to the code in the real-valued case, is there a performance benefit to having just:
```C++
out0 = in0_type(rint(in0.real()), rint(in0.imag())
```
within the `in1 == 0` branch instead?